samples/15.b.incoming-activity-event/README.md

```diff
@@ -0,0 +1,23 @@
+# Sample - Forward all incoming activity to JavaScript event
+
+Web Chat client that will forward all incoming activity to JavaScript event named `webchatincomingactivity`.
+
+# Test out the hosted sample
+
+- [Try out MockBot](https://microsoft.github.io/BotFramework-WebChat/15.b.incoming-activity-event)
+
+# Things to try out
+
+- Open development console in your browser
```

Comment:
Should we stay consistent and use the same text for every sample?

Reply:
Yes, let's make a PR to scrub all the text in README.